test/csuite/wt7989_compact_checkpoint/main.c

```diff
@@ -162,11 +163,15 @@ run_test(const char *home, const char *uri, bool stress_test)
     testutil_check(session->close(session, NULL));
 
     /* Check if there's at least 10% compaction. */
-    testutil_assertfmt(file_sz_before * 0.9 > file_sz_after,
-      "Compact failed to reduce file size by at least 10%%\n"
-      "Compressed file size MB: %f\n"
-      "Original file size MB: %f\n",
+    printf("Compressed file size MB: %f\nOriginal file size MB: %f\n",
       file_sz_after / (1024.0 * 1024), file_sz_before / (1024.0 * 1024));
+
+    /*
```

Comment:
Should we add a FIXME-ticketID here with the ticket that fixes the problem ?

Reply:
Done, thanks!